Networking/Networking/Model/Product/Product.swift

```diff
@@ -711,6 +711,11 @@ public struct Product: Codable, GeneratedCopiable, Equatable, GeneratedFakeable
 
         // Attributes
         try container.encode(attributes, forKey: .attributes)
+
+        // Subscription
+        if productType == .subscription || productType == .variableSubscription {
+            try container.encode(subscription, forKey: .metadata)
+        }
```

Comment:
Nit: Should we add a helper method to build metadata and send it to the encoder instead of sending subscription info directly? This will come in handy when there is any other data besides subscription info we need to send to metadata.